### ISSUE TITLE
Бизнес-процессы при взаимоотношениях с клиентами

### ISSUE THREAD

**Issue by SvetlanaRes** [Contabilizare/c5#172]
Дабы структурировать потребности и возможности системы хотелось бы прописать возможные этапы процесса взаимоотношения с клиентом в разрезе документов в систем к5:
1) **при работе с клиентом по предоплате**:
выставляется счет на оплату / оприходованы денежки / произведены работы / сформировано оказание услуги / выписана НН
в разрезе документов к5:
Заказ покупателя (ПФ Счет на оплату) / Оплата (наличные/безналичные) / Запись времени (предварительно Проект) / Реализация* / НН

*в данном процессе не ясно формирование документа Реализации - он заполняется и на основании Заказа покупателя и на основании Записи времени. Что не так и где указано? Т.е. заказ должны формировать, но в данном случае проводить реализацию на основании Записи времени.

2) **при работе с клиентом по факту выполненных работ**:
производятся работы / фиксируется оказание услуги (с НН) / должен быть выписан счет на работу / оприходованы ДС
в разрезе документов к5:
Запись времени (предварительно Проект) / Реализация* (НН) / заказ покупателя или счет на основании?** / Оплата

* выдает сообщение: 
![image](https://user-images.githubusercontent.com/82224240/124918021-342cc000-dffd-11eb-8b25-a570dddd1e73.png)
что пошло не так?

** не вижу как сформировать ПФ счет на оплату и/или Заказ покупателя?

**Comment by Nullarity** [comment]
@SvetlanaRes для первого варианта, как лучше реализовать? Дело в том, что при вводе на основании ЗВ, предусматривается сценарий отработано-оплачено (слесари-сантехники, электрики, консультации и тому подобное). Но у там у тебя готовится инфа по #114, и такой процесс в твоем случае не подойдет. Поэтому пока видимо РН надо вводить на основании заказа?

**Comment by Nullarity** [comment]
@SvetlanaRes На каком релизе ты моделируешь ситуации, на последнем?

**Comment by SvetlanaRes** [comment]
> @SvetlanaRes для первого варианта, как лучше реализовать? Дело в том, что при вводе на основании ЗВ, предусматривается сценарий отработано-оплачено (слесари-сантехники, электрики, консультации и тому подобное). Но у там у тебя готовится инфа по #114, и такой процесс в твоем случае не подойдет. Поэтому пока видимо РН надо вводить на основании заказа?

да, разбираю/готовлю инфу по #114 и столкнулась с этими вопросами.

**Comment by SvetlanaRes** [comment]
> @SvetlanaRes На каком релизе ты моделируешь ситуации, на последнем?

на 14 релизе
